Nautilus/Handlers/ModDataBankHandler.cs

```diff
@@ -0,0 +1,111 @@
+using BepInEx;
+using Nautilus.Utility;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+
+namespace Nautilus.Handlers;
```

Comment:
Class should be static

Reply:
ah, you're right, was actually just thinking about that haha.

Reply:
fixed.